workbench/public/app.scss

```diff
@@ -16,13 +16,9 @@
 @import '../node_modules/@elastic/eui/src/global_styling/variables/colors';
 @import '../node_modules/@elastic/eui/src/global_styling/variables/size';
 
-.sql-console-page-header {
-  padding: $euiSizeS $euiSizeL;
-  font-weight: 600;
-}
-
 .sql-console-query-container {
   padding: $euiSizeL;
+  height: 1142px;
```

Comment:
I removed this line and line 57 in #741 , are heights here necessary? It's causing double vertical scroll bars.

Reply:
probably because I didn't sync the code base, will remove it